### PR TITLE
Setup docs

### PR DESCRIPTION
**Pull request by Kodylow**
- Adds mutiny setup documentation
- Updates docker-compose files to use fedimint 0.1 instead of release candidate

### CODE REVIEW

**Review by dpc — DISMISSED** [review]

**Comment by dpc** [comment]
[Rendered](https://github.com/Kodylow/fedimint/blob/38c9e2ca4e0ae10e548713c97d32c7cd3ca60893/docs/setup-docs.md)

File: docs/setup-docs.md [hunk 1]
```diff
@@ -0,0 +1,463 @@
+# Setting up a Federation + Lightning Gateway on Mutinynet
+
+This is a guide for setting up a 3/4 federation + lightning gateway on mutinynet, a test network for bitcoin with 30 second block times. You can run this same setup on a different test network or on mainnet with real bitcoin, but that is not encouraged while fedimint remains under active development.
+
+```diff
+- **WARNING** Fedimint 0.1 is alpha software released under an MIT License, do not use it with real bitcoin (yet), only use regtest or test networks like mutinynet.
+```
```
Comment (elsirion) [inline docs/setup-docs.md:7]:
```suggestion
> [!WARNING]  
> Fedimint 0.1 is alpha software released under an MIT License, do not use it with real bitcoin (yet), only use regtest or test networks like mutinynet.
```

Will look like ([docs here](https://github.com/orgs/community/discussions/16925)):

> [!WARNING]  
> Fedimint 0.1 is alpha software released under an MIT License, do not use it with real bitcoin (yet), only use regtest or test networks like mutinynet.

File: docs/setup-docs.md [hunk 2]
```diff
@@ -0,0 +1,463 @@
...
+
+<p align="center">
+<img src="setup-docs-assets/received.png" alt="Received" width="500">
+</p>
+
+Your federation gateway is now ready to provide lightning services to the federation! You should be able to send and receive lightning payments to users of the federation via the gateway, give it a try using a compatible client like the fedimint-cli or [webimint-rs](https://github.com/fedimint/webimint-rs) (demo at https://webimint.sirion.io/) ,
+
+```diff
+- **WARNING** webimint does not have persistent storage so you'll lose your funds if you close the browser tab. Do not use it with real bitcoin.
+```
```
Comment (elsirion) [inline docs/setup-docs.md:463]: Same nit as above, I doubt what you wanted is a diff view.

**Review by elsirion — APPROVED** [review]

**Comment by elsirion** [comment]
Just skimmed over it, but if anything is wrong people will tell us.

**Comment by fedimint-backports** [comment]
Backport failed for `releases/v0.1`, because it was unable to cherry-pick the commit(s).

Please cherry-pick the changes locally.
```bash
git fetch origin releases/v0.1
git worktree add -d .worktree/backport-3147-to-releases/v0.1 origin/releases/v0.1
cd .worktree/backport-3147-to-releases/v0.1
git checkout -b backport-3147-to-releases/v0.1
ancref=$(git merge-base 1768fba076537c11d65fa79958b442907b1b36c3 5b8993d59cd46212f4606654afa642e07002db60)
git cherry-pick -x $ancref..5b8993d59cd46212f4606654afa642e07002db60
```

**Comment by elsirion** [comment]
Do we need to backport this? Seems like something that is sufficient on master. Few will actually check out the v0.1 branch to run it imo.

**Comment by justinmoon** [comment]
I think it's fine to just have on master

**Pull request merged**